server/db/models/Menu.js

```diff
@@ -0,0 +1,27 @@
+const {
+    Model,
+    DataTypes,
+  } = require('sequelize');
+  const db = require('../db');
+  
+class Menu extends Model {}
+
+Menu.init(
+{
+id: {
+    type: DataTypes.UUID,
+    defaultValue: DataTypes.UUIDV4,
+    primaryKey: true,
+    },
+name: {
+    type: DataTypes.STRING, // is menu name just venue name?
+    allowNull: false,
+    validate: { 
+        notEmpty: true,
+    },
```

Comment:
I think we talked about maybe the menu could be a way for restaurants to categorize their offerings, so one venue could have a Wine menu and a Beer menu and a Cocktail menu that users could browse individually

Reply:
yes we did! I'll fix the other thing and remove the comment here.